test/box/gh-6040-hash-not-tracked-by-mvcc.lua

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env tarantool
```

Comment:
It's better to use common tx_man.lua. Perhaps we should give it different, more common name, but I think it's not very important.

Reply:
Fixed.